go.mod

```diff
@@ -255,9 +254,10 @@ replace (
 	github.com/go-kit/log => github.com/go-kit/log v0.1.0
 	github.com/go-logfmt/logfmt => github.com/go-logfmt/logfmt v0.5.0
 	github.com/go-logr/logr => github.com/go-logr/logr v0.4.0
-	github.com/go-openapi/jsonpointer => github.com/go-openapi/jsonpointer v0.19.3
-	github.com/go-openapi/jsonreference => github.com/go-openapi/jsonreference v0.19.3
-	github.com/go-openapi/swag => github.com/go-openapi/swag v0.19.5
+	github.com/go-openapi/jsonpointer => github.com/go-openapi/jsonpointer v0.19.5
+	github.com/go-openapi/jsonreference => github.com/go-openapi/jsonreference v0.19.5
+	github.com/go-openapi/spec => github.com/go-openapi/spec v0.20.3
```

Comment:
I expect `hack/lint-dependencies.sh` would show this should be dropped as well

Reply:
Ran hack/lint-dependencies.sh and handled the following:

```
$ hack/lint-dependencies.sh

Use the given commands to remove pinned module versions that aren't actually used:
GO111MODULE=on go mod edit -dropreplace cloud.google.com/go/datastore
GO111MODULE=on go mod edit -dropreplace cloud.google.com/go/pubsub
GO111MODULE=on go mod edit -dropreplace github.com/google/martian
GO111MODULE=on go mod edit -dropreplace github.com/go-openapi/spec
GO111MODULE=on go mod edit -dropreplace rsc.io/quote/v3
GO111MODULE=on go mod edit -dropreplace rsc.io/sampler
All pinned versions of checked dependencies match their preferred version.
```